src/Common/Exceptions/BaseException.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __BASE_EXCEPTION_H_
+#define __BASE_EXCEPTION_H_
+
+#include <iostream>
+
+namespace FileManager {
+
+    class BaseException {
+    public:
+
+        std::string message;
```

Comment:
Can we make this `private`?

Reply:
Mmmm we can, but I believe this should be accessible to anyone, may through a getter.